dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorLlvm.java

```diff
@@ -394,6 +394,8 @@ public Expression visitCallInst(CallInstContext ctx) {
             //FIXME ignore side effects of inline assembly
             if (resultRegister != null) {
                 block.events.add(newLocal(resultRegister, program.newConstant(returnType)));
+            } else {
+                visitChildren(ctx);
```

Comment:
Just call `ctx.inlineAsm().accept(this)` to traverse down. 

Also, we should probably log a warning when executing the first branch because we interpret the inline assembly as just some unconstrained value.